packages/x-components/src/x-modules/experience-controls/store/module.ts

```diff
@@ -3,4 +3,25 @@
  *
  * @internal
  */
-export const experienceControlsXStoreModule: any = {};
+export const experienceControlsXStoreModule: any = {
```

Comment:
You should type the module as `ExperienceControlsXStoreModule` importing the type file that you are creating in this PR